posthog/models/property/util.py

```diff
@@ -591,6 +594,15 @@ def get_single_or_multi_property_string_expr(
     return f"{expression} AS {query_alias}"
 
 
+def normalize_url_breakdown(breakdown_value, breakdown_normalize_url: bool = True):
```

Comment:
nit: This function feels a bit awkward, where the second parameter controls whether it does anything at all.

Might be clearer to make this function always add the extra SQL, and control at one level up whether to call it or not?

Reply:
This is tricky. It's called in more than one place. So, if you lift the conditional out of the method there's an increased risk of the conditional choice drifting as people change it...

But it is yucky to pass this boolean around so much 🦏 🤔

Reply:
Ahhh, I see. Yeah, problematic 🤔